### ISSUE TITLE
ansible stalls at "Add Repo to `apt-get`"  on Ubuntu 18.10 + fix

### ISSUE THREAD

**Issue by bbtdev** [FlorianKempenich/ansible-role-erlang-elixir-phoenix#1]
on Ubuntu 18.10 dpkg -i opens a GUI and ansible stalls, we need to run:
`DEBIAN_FRONTEND=noninteractive sudo dpkg -i erlang-solutions_1.0_all.deb`

in ansible:
```
    - name: "Add Repo to `apt-get`"
      environment: 
        DEBIAN_FRONTEND: noninteractive
      command: dpkg -i "{{ erlang_repo_package }}" chdir="{{ erlang_repo_temp_dir }}"
      become: yes
```

**Comment by HelloThisIsFlo** [comment]
Oh Thanks a lot!
I didn't notice yet. I'm only running on the LTS versions and frankly .... I could do better with integration testing ... given it's inexistent at that point ^_^
Do you want to submit a merge request, or you prefer I update the file myself?

**Comment by bbtdev** [comment]
@FlorianKempenich never contributed to a public repo before, so I will tomorrow :).
Always wanted to contribute.

**Comment by bbtdev** [comment]
@FlorianKempenich I got a permission denied, something is wrong with my git. Sorry, you have to do it yourself.

**Comment by HelloThisIsFlo** [comment]
Hey @bbtdev,

I could definitely fix it myself in no time. But I believe helping you contribute for the very first time to a public repo is a much more interesting thing to do ;)


There's nothing wrong with your `git`. You're not supposed to be able to directly push to public repos. Think about it this way, if everyone could add a line of code to, say, the Linux Kernel we'd have serious security issues.

Instead, there's a mechanism called a **Pull-Request** on Github (Merge-Request on Gitlab) that allow you to contribute to public repos. Here is how it works:

1. You `Fork` the project
    * This means creating a **copy** of the project in your own account
    * To do this, it's simple: Click on the `Fork` button on the top right of the project page in Github
2. You'll now end up on your very own 'Fork' of the project
    * It'll be named:`bbtdev/ansible-role-erlang-elixir-phoenix`
3. You `pull` **this new repo `bbtdev/ansible-role-erlang-elixir-phoenix`**
4. You make the changes.
5. You push. 
    * You now can because it's **your** repo, you're only pushing to `bbtdev/ansible-role-erlang-elixir-phoenix`
6. You go on the github page of your 'Fork'
7. You click on `Pull Requests -> New pull request`
    * This will basically ask the owner of the **original repo** (me in our case) if it's cool for them to __pull__ the changes from **your repo**
    * It'll open a space for discussing the changes, the reasons for the changes, and potentially tweaking it a little bit to fit the standard of the original repo.
8. Once the discussion is settled, and both the owner and the contributor are happy, the change can be pulled in the original project.
9. Enjoy your first contribution to a public (albeit modest) open-source project :)

If you have any questions, hit me up ;)

**Comment by bbtdev** [comment]
Hey @FlorianKempenich 

Ha, thanks for the detailed instructions. At work I just open branches, I did not know about this flow. Thank you :)

**Comment by HelloThisIsFlo** [comment]
Congrats on your first public contribution 🎉🎉
Happy to see you went through it without problems 😃

_Tip: You can also reference the Merge Request directly in the issue, like this: #2_

**Comment by HelloThisIsFlo** [comment]
Actually it's the other way around, if **this** issue is referenced in the _Pull Request_, then once the _Pull Request_ is merged, the issue is automatically closed.

See? I'm learning as well ^_^
